<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph</class>
 <widget class="QWidget" name="Graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 1px solid black;</string>
  </property>
  <widget class="QCustomPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>591</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_xmin">
      <property name="text">
       <string>    Xmin</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_xmax">
      <property name="text">
       <string>    Xmax</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="x_min">
      <property name="text">
       <string>-10</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="x_max">
      <property name="text">
       <string>10</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="build_graphics">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>390</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>build</string>
   </property>
  </widget>
  <widget class="QLabel" name="expression">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>sinx</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_ymin">
      <property name="text">
       <string>    Ymin</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_ymax">
      <property name="text">
       <string>    Ymax</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="y_min">
      <property name="text">
       <string>-1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="y_max">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
